### PR TITLE
travis: notify maintainers via mail about builds

### PR DESCRIPTION
**Pull request by miri64**
Inspired by https://github.com/RIOT-OS/Tutorials/pull/60

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ before_install:
   - sudo apt-get install coreutils cppcheck pcregrep python3 python3-pip uncrustify
   - sudo pip3 install flake8
 
+notifications:
+  email:
+    recipients:
+    - maintainers@riot-os.org
```
Comment (smlng) [inline .travis.yml]: list is actually maintainer@riot-os.org (singular)

Comment (miri64) [reply .travis.yml]: Fixed

**Comment by smlng** [comment]
@miri64 can you have a look at #65, to fix master, thx

**Comment by miri64** [comment]
I guess we need to add travis to the whitelist for maintainers as well, right?

**Comment by miri64** [comment]
I hope you used `Squash on top of master` when merging this ;-).

**Pull request merged**